frontend/app/src/components/accounts/blockchain/WalletAddressesImport.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+defineProps<{
+  disabled: boolean;
+}>();
+
+const emit = defineEmits<{
+  (e: 'update:addresses', addresses: string[]): void;
+}>();
+
+const { t } = useI18n();
+const [DefineButton, ReuseButton] = createReusableTemplate<{ buttonDisabled: boolean }>();
+
+const { isPackaged, metamaskImport } = useInterop();
+
+async function importAddresses() {
+  const addresses = await metamaskImport();
+  emit('update:addresses', addresses);
```

Comment:
Can you try/catch here too? I see a warning about an unhandled exception on timeout.